src/react/account/AccountUserAccessKeys.jsx

```diff
@@ -43,6 +48,41 @@ const CreatedOnCell = rowValue => {
   );
 };
 
+const ToggleAccessKeyStatus = rowValue => {
+  const { accessKey, status: accessKeyStatus } = rowValue;
+  const IAMClient = useIAMClient();
+  const { IAMUserName } = useParams();
+  const loading = useSelector(
+    (state: AppState) => state.networkActivity.counter > 0,
+  );
```

Comment:
I'm not sure the loading status is useful here

Reply:
Loading deleted -> done